### PR TITLE
Update to expand-video.js (video hover changes) and fix for portrait mode laptop

### PR DESCRIPTION
**Pull request by anonish**
Updated the tablet mode code, apparently width of 600 was still not allowing QR to load in portrait.  400 works and tested ok on my tablet/laptop

>previous fix in https://github.com/ctrlcctrlv/infinity/commit/e115bff930251d1370c5cc9293ef930f3a8ca1a1

### CODE REVIEW

**Comment by anonish** [comment]
I'm attempting to create a "hover follows mouse" implementation on videos (as per 8chanX's image hover code).
Changing line 128 to "mousemove" will create this behavior, however due to the nature of infinity's code, it's very cpu expensive, so instead I left it at "mouseover" which does the following:
* the video opens at the mouse location (on top of the thumbnail) and if the size is too large, fits into the current window width/height
* moving out of the thumbnail unhovers (as per the original code)
* if at the bottom of the screen, the video opens above the current location to fit into the screen, but always maintaining X coordinate on top of the thumbnail

enabling "mousemove" causes the same behavior, only the video will dynamically resize as you move the mouse, staying within the window bounds (and again on top of the thumbnail)

the other features (sound when moving scrollwheel, clicking to expand) remain.  as a bonus it also fixes the "no hover after unexpanding a previously expanded video" (which is still broken for images in 8chanx)

please implement this, either as an option or as default.  I've never quite understood why the hover code created the video on the top right corner to begin with (and both infinity and 8chanx did this behavior originally).  for certain setups (widescreen browsing in particular) this can be extremely annoying, useless (with small images

**Comment by anonish** [comment]
On index pages, restore original behavior (temp fix)
On thread pages, fixed multiple file posts so all posts hover
- still need to fix width for multiple posts as the ones on the right side of the page extend beyond the page (on original method they become tiny)
- fix catalog

long term i am thinking of just redoing the whole hover file (unless someone else does first).

File: inc/template.php [hunk 1]
```diff
@@ -22,7 +22,8 @@ function load_twig() {
 	$loader = new Twig_Loader_Filesystem($config['dir']['template']);
 	$loader->setPaths($config['dir']['template']);
 	$twig = new Twig_Environment($loader, array(
-		'autoescape' => false,
+        'autoescape' => false,
+        'auto_reload' => true,
```
Comment (ctrlcctrlv) [inline inc/template.php]: why are you editing the twig environment? please don't do that ;_;

**Comment by anonish** [comment]
>why are you editing the twig environment? please don't do that ;_;
that wasnt me! that was from the merge https://github.com/anonish/infinity-anonish/commit/b7a6f6419f2a20ea0f9b96920433a963d7904ce6
i dont know how to remove those extra commits, i just wanted the expand-video.js ones

**Comment by ctrlcctrlv** [comment]
OK. Is this compatible with #401? Do you intend for both to be merged?

**Comment by anonish** [comment]
>OK. Is this compatible with #401? Do you intend for both to be merged?

no, i added https://github.com/ctrlcctrlv/infinity/pull/401 as a temp fix because i was getting annoyed at the webms hiding on top.  I'll cancel that pull request :skull:

**Pull request merged**